release/7-2/ubuntu18.04-arm32v7/meta.json

```diff
@@ -22,5 +22,5 @@
     "EndOfLife": "2023-04-02",
     "DistributionState": "Validated",
     "UseInCi": false,
-    "Architecture": "arm64"
+    "Architecture": "arm32"
```

Comment:
This is more correctly, the host architecture and we don't have an ARM32 host.  
We either need to translate this to ARM64 when choosing a host or add another field.

Reply:
I think I made the changes to make this work for CI and image build, test and release.